### PR TITLE
Update junglespawn command

### PR DESCRIPTION
**Pull request by xxx3456789**
+ Feature :
!junglespawn 1(Only normal camps) / 2 (Only Epic monsters) / 3 (normal + Epic)
- Issues :
They don't have AI, like woods standing there.

### CODE REVIEW

**Review by danil179 — CHANGES_REQUESTED** [review]
Too much hardcoded things. That should be generated from some file and without using hardcoding.

File: GameServerLib/Chatbox/Commands/JunglespawnCommand.cs [hunk 1]
```diff
@@ -17,9 +18,412 @@ public JunglespawnCommand(ChatCommandManager chatCommandManager, Game game)
...
+                default:
+                    ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.SYNTAXERROR);
+                    ShowSyntax();
+                    break;
+            }
+            //_logger.Warn($"{ChatCommandManager.CommandStarterCharacter}{Command} command not implemented");
+            //ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.INFO, "Command not implemented");
+        }
+
+        private void BlueTeam01()
```
Comment (FurkanS1821) [inline GameServerLib/Chatbox/Commands/JunglespawnCommand.cs:63]: These function names don't really look self-explanatory, can we change them with something like `Spawn(Blue|Red)(Left|Right)Jungle` and `Spawn(Dragon|Baron)` please?

File: GameServerLib/Chatbox/Commands/JunglespawnCommand.cs [hunk 2]
```diff
@@ -17,9 +18,412 @@ public JunglespawnCommand(ChatCommandManager chatCommandManager, Game game)
...
+        {
+            var BlueWolfBig = new Monster(
```
Comment (FurkanS1821) [inline GameServerLib/Chatbox/Commands/JunglespawnCommand.cs:65]: local variables should be `likeThis`

File: GameServerLib/Chatbox/Commands/JunglespawnCommand.cs [hunk 3]
```diff
@@ -17,9 +18,412 @@ public JunglespawnCommand(ChatCommandManager chatCommandManager, Game game)
...
+                    RedTeam02();
+                    Dragon();
+                    Baron();
+                    break;
+                default:
+                    ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.SYNTAXERROR);
+                    ShowSyntax();
+                    break;
+            }
+            //_logger.Warn($"{ChatCommandManager.CommandStarterCharacter}{Command} command not implemented");
```
Comment (FurkanS1821) [inline GameServerLib/Chatbox/Commands/JunglespawnCommand.cs:59]: Please remove unused code instead of commenting it out.

**Review by FurkanS1821 — CHANGES_REQUESTED** [review]
I don't know if adding a command to spawn mindless jungle monsters is a step forward but let's go

**Comment by lzardy** [comment]
This PR has been superseded by #1269

**Pull request closed without merging**